### ISSUE TITLE
Version 1.6 high CPU usage / 1.7 no DS4 V2 detection

### ISSUE THREAD

**Issue by dukzz** [nefarius/ScpToolkit#944]
Greetings!

I was using ScpToolkit v1.6.238.16010 (The latest one that is not BETA) with my Dualshock 4 V2 via USB cable and everything worked fine - until the day I connected my controller via flawed USB cable and Windows started installing a new driver for it. After plugging in another USB cable that works fine - scp toolkit did not recognize it anymore, so I used the Driver Wiper, reinstalled the drivers and the controller worked, but now the ScpService.exe uses 25% of my CPU (i5 6500 3.2ghz).
So I tried the latest version ScpToolkit v1.7.277.16103-BETA, but it does not detect my Dualshock 4 V2 after wiping all the drivers and everything.
Can anyone who had this issue before please give some advice on fixing the high CPU usage with the 1.6 or has any other solution?

TL;DR
- ScpToolkit v1.6.238.16010 (The latest one that is not BETA) - Dualshock 4 V2 via USB makes the ScpService.exe use 25% of my CPU  (i5 6500 3.2ghz), Win7 64bit.
- Latest version ScpToolkit v1.7.277.16103-BETA - does not detect my Dualshock 4 V2 after wiping the drivers.

Please help and thank you in advance!
